sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ImplementISerializableCorrectly.CSharp9.cs

```diff
@@ -0,0 +1,282 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Tests.Diagnostics
+{
+    [Serializable]
+    public record SerializableRecord : ISerializable
+    {
+        public SerializableRecord()
+        { /*do something*/ }
```

Comment:
I know it's a nitpick, but by removing the `/*do something*/` comment and by bringing the brackets on the same line with the method declaration, you'd remove some clutter from the test cases

```
public SerializableRecord() {}
protected SerializableRecord(SerializationInfo info, StreamingContext context) {}
public virtual void GetObjectData(SerializationInfo info, StreamingContext context) {}
```
is much better